sites/melbourne/lib/melbourne/github/issue_fetcher.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Melbourne
+  module Github
+    class IssueFetcher
+      def initialize(github_token:, repo_owner:, repo_name:)
+        @github_token = github_token
+        @repo_owner = repo_owner
+        @repo_name = repo_name
+      end
+
+      def fetch_all
+        issues = fetch_issues
+        group_by_milestone(issues)
+      end
+
+      private
+
+      attr_reader :github_token, :repo_owner, :repo_name
+
+      def fetch_issues # rubocop:disable Metrics/AbcSize
```

Comment:
Same as above; these classes are just data processing scripts not meant to live forever or to be used regularly at all